web/js/pages/Home.jsx

```diff
@@ -17,26 +22,42 @@ type Props = {
 }
 
 class Home extends React.Component < Props > {
+  componentWillMount() {
+    document.body.style.margin = 0;
+    document.body.style.backgroundColor = '#F5F5F5';
+  }
```

Comment:
Can we just move this to index.js? Not sure if this is the best place to put it.